Services/BrokerService/Controllers/BrokerController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using BrokerServices;
+using BrokerServices.Utils;
+using Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BrokerService.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class BrokerController : ControllerBase
+    {
+        [Route("getAllCurrencies")]
+        [HttpGet]
+        public List<IMarketInstrument> GetAllCurrencies([FromBody] BankType bankType)
```

Comment:
Is this okay to have body in get requests?

Reply:
fixed